.github/workflows/ci.yaml

```diff
@@ -24,15 +31,17 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ github.ref_name }}
     needs: check
-    if: needs.check.result == 'success'
+    if: needs.check.result == 'success' && startsWith(github.ref, 'refs/tags/')
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
-      - name: Extract VERSION from tag
-        run: echo "VERSION=${GITHUB_REF#refs/tags/v}" >> $GITHUB_ENV
```

Comment:
This removal caused the job to fail, I'll restore it.

Reply:
Actually, I see now that you moved it to the top env section. However, the v prefix needs to be stripped from it.

Reply:
What! how!

Reply:
2748254

Reply:
AHhh is the `v`

Reply:
wait

Reply:
Code already available from new.yaml :)

Reply:
Not sure what you meant by that... but feel free to send another PR. Another topic: previously the archives only contained the shellcheck binary, but now they also contain README and LICENSE.

I have no strong opinion about it. On VS Code Shellcheck they would work either way:

https://github.com/vscode-shellcheck/vscode-shellcheck/blob/master/bindl.config.js

As we selectively only extract the binary from the archive.

Just wanted to know if this change was intentional or not.

Reply:
Oh, and btw the archives were published!

https://github.com/vscode-shellcheck/shellcheck-binaries/releases/tag/v0.9.0

Reply:
Since the main objective is to repackage all archives into `.tar.gz` I think it is better to maintain the same structure/content. Maybe for others using the archives is useful. The only exception is the Darwin arm64 but you know... Apple Silicon

Reply:
> Oh, and btw the archives were pu

Awesome! 🤯😎

Reply:
@felipecrs Sorry but can you also upload the windows archives for v0.8.0. Thanks 🤯🥳🙏

Reply:
Sure, done!